### PR TITLE
Userguide instruction fixes

### PR DESCRIPTION
**Pull request by JonLamy**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -87,9 +98,29 @@ Examples:
 * `add finish problem set 5 by/tomorrow 5pm m/CS2040S`
 * `add finish SEP application by/2022-12-25 23:59`
 
+<div markdown="block" class="alert alert-primary">
+
+:bulb: Deadline Formats
+
+CodeConnect uses the [jchronic] natural language date parser. This means that natural language inputs such as "tomorrow" or "next sunday" will be understood by CodeConnect!
+
+For more information on valid deadline formats, check the jchronic documentation [here](#).
```
Comment (parnikkapore) [inline docs/UserGuide.md:107]: Where the URL?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -232,10 +296,17 @@ Example:
...
 
 Format: `clear`
 
+<div markdown="block" class="alert alert-warning">
+
+:exclamation: Warning!<br>
+Deleted contacts are **unrecoverable**!
+
+</div>
+
```
Comment (parnikkapore) [inline docs/UserGuide.md:309]: Good catch!

**Review by parnikkapore — COMMENTED** [review]
Apparently the Deadline Formats section is missing the URL...

**Review by danzzzerl — APPROVED** [review]
LGTM!

**Pull request merged**